Library/Formula/clfft.rb

```diff
@@ -0,0 +1,48 @@
+class Clfft < Formula
+  homepage "https://github.com/clMathLibraries/clFFT/"
+  url "https://github.com/clMathLibraries/clFFT/archive/v2.4.tar.gz"
+  sha256 "d77506af774bbe8ccf4226a58e623c8a29587edcf02984e72851099be0efe04b"
+
+  depends_on "cmake" => :build
+  depends_on "boost" => :build
+
+  patch do
+    # rename Client
+    url "https://github.com/clMathLibraries/clFFT/pull/53.diff"
+    sha1 "8f9f66419c3add6beb1def058884dc12822313f2"
+  end
+
+  patch do
+    # don't install py files in bin
+    url "https://github.com/clMathLibraries/clFFT/commit/ae845846990bfabe5c01ee8629b8df32ca9ce7a9.diff"
+    sha1 "6a28e3347eb5e849fd63eccd0d1750455af437cd"
+  end
+
+  patch do
+    # properly deal with rpaths
+    url "https://github.com/clMathLibraries/clFFT/pull/59.diff"
+    sha1 "2f0929f0aff6ea87d799d0e7183c3bff58dd9098"
+  end
+
+  patch do
+    # don't use lib64 in lib path
+    url "https://github.com/clMathLibraries/clFFT/pull/61.diff"
+    sha1 "b4cec7072c592a68c5c603d6f2e628baaedca35e"
+  end
+
+  patch do
+    # install cmake config and version files
+    url "https://github.com/clMathLibraries/clFFT/pull/68.diff"
+    sha256 "31e5a6ffc8e94e30970035162c2b45418795929402513b189d2f8fd3a3810ef0"
+  end
```

Comment:
They have all been accepted too. I prepared the formulas before they were accepted. I'll replace them with the actual commits.